app/test_app/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
```

Comment:
I guess the test_app folder was accidentally created using `flutter create test_app`. Please delete this folder.

Reply:
deleted